src/components/RightPane/Map/UCIMap.tsx

```diff
@@ -1,7 +1,7 @@
 import '../../../../node_modules/leaflet.locatecontrol/dist/L.Control.Locate.min.js';
```

Comment:
```suggestion
```
Since you're in this file, could you try removing this line and see if the map still works? I'm pretty sure this is what the `import leaflet.locatecontrol` line is supposed to do

Reply:
Yeah it works! Deleting this line is safe